Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -80,33 +69,35 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
       |> \.dataSource .~ self.dataSource
       |> \.delegate .~ self
       |> \.tableHeaderView .~ self.projectNavigationSelectorView
-      |> \.tableFooterView .~ UIView(frame: .zero)
+      |> \.tableFooterView .~ UIView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: 1))
```

Comment:
This gets rid of the line at the bottom of the tableview on each of the tabs. I thought setting this to an empty view would work but it was still there until I did this. Let me know if you know of a cleaner alternative.

Reply:
Saved the height value (1) into the styles enum at the top and changed the width to 0.